index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute Page</title/>
+     <link rel="stylesheet" href="styles.css" />
+    </head>
+    <body>
+    <main id="main">
+      <h1 id="title">Tribute Page</h1> 
+      <div id= img-div>
+        <img id="image" src="https://th.bing.com/th/id/OIP.-fgIcBl7BmMJwFdXPdPpzQHaE7?pid=ImgDet&rs=1" alt="youtube logo">
+        <figcaption id="img-caption">Youtube logo</figcaption>

```

Comment:
its advisable to place all links inside the nav tag. this makes the code to look clean.